src/metax_api/models/catalog_record.py

```diff
@@ -231,6 +231,13 @@ class CatalogRecord(Common):
 
     data_catalog = models.ForeignKey(DataCatalog, on_delete=models.DO_NOTHING, related_name='records')
 
+    state = models.CharField(
+        max_length=200,
+        null=False,
+        default='published',
+        help_text='Publishing state (published / draft) of the dataset.'
+    )
```

Comment:
``state`` field should only values from a list of pre-determined values. See how the "choices" option is being used in other fields, for example in ``cumulative_state``. Allowed values should be ``"draft"`` and ``"published"``.